### DISCUSSION TITLE
Assign a special name to the last console output.

### DISCUSSION THREAD

**Discussion by maxslimmer** [microsoft/debugpy#1789] [Enhancements]
Feature Request

I would be cool if you could access the last (and maybe second-to-last) values returned from debug console evaluations via a special name (like maybe "_" and "__"), similar to ipython and other interactive REPL.

```
> 2
2

> 1
1

> f"last expression: {_}, second-to-last: {__}"
'last expression: 1, second-to-last: 2'
```

**Comment by karthiknadig** [comment]
Debug console itself is handled by VS Code. Moving this to the right repo for further discussion.

**Comment by vs-code-engineering** [comment]
This feature request is now a candidate for our backlog. The community has 60 days to [upvote](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) the issue. If it receives 20 upvotes we will move it to our backlog. If not, we will close it. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by roblourens** [comment]
I think I would send this back to you @karthiknadig, having a special value like this would have to be something that the debug adapter would do, does that make sense? VS Code can't change how a command is evaluated

**Comment by karthiknadig** [comment]
@roblourens Thanks for clarifying, I read this completely wrong when I moved it. This should go to `debugpy`.

**Comment by rchiodo** [comment]
Thanks for the suggestion. Moving to a discussion item for up votes.
